tests/src/test/java/me/snowdrop/istio/dt/OpenshiftIT.java

```diff
@@ -0,0 +1,134 @@
+package me.snowdrop.istio.dt;
+
+
+import io.fabric8.openshift.api.model.v3_1.Route;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import me.snowdrop.istio.api.model.IstioResource;
+import org.arquillian.cube.istio.impl.IstioAssistant;
+import org.arquillian.cube.openshift.impl.client.OpenShiftAssistant;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static org.awaitility.Awaitility.await;
+
+@RunWith(Arquillian.class)
+public class OpenshiftIT {
+    @ArquillianResource
+    private IstioAssistant istioAssistant;
+
+    @ArquillianResource
+    private OpenShiftAssistant openShiftAssistant;
+
+    private static String istioURL;
+    private static String jaegerURL;
+    private static boolean isInit = false;
+    private static List<IstioResource> routeRule = null;
+
+    private final String appUrl = "greeting/";
```

Comment:
Perhaps have this capitalized?

Reply:
I don't get it. How you imagine it would look like?

Reply:
`appUrl` -> `APP_URL`

Reply:
Capitals are normally used for static. So this should either stay as is or become static and in capitals.

Reply:
But of course I'm too late :)

Reply:
Indeed it should be static, missed that :). I will add it